ait/dsn/sle/common.py

```diff
@@ -97,6 +97,8 @@ class SLE(object):
 
     def __init__(self, *args, **kwargs):
         ''''''
+        self. _downlink_frame_type = ait.config.get('dsn.sle.downlink_frame_type',
+                                      kwargs.get('telemetryframetype', 'TMTransFrame'))
```

Comment:
The kwargs key to check for here would be `downlink_frame_type` as well, not `telemetryframetype`.